build.gradle

```diff
@@ -211,6 +211,14 @@ subprojects {
             csv.enabled = false
         }
     }
+
+    if (!path.startsWith(':cloudfoundry-identity-samples')) {
+        task collectDependencies(type: Copy) {
+            from configurations.default
+            into "${rootProject.buildDir}/dependencies"
+            exclude 'cloudfoundry-identity-*.jar'
```

Comment:
Do we really need to exclude uaa jars from dependencies? Are they really included otherwise?

Reply:
`uaa.war` uses `service.jar`, `model.jar`, etc. So they do show up as dependencies with "Unknown license". That's why I excluded them.